admin/common_auth/admin.py

```diff
@@ -11,7 +11,7 @@ class PermissionAdmin(admin.ModelAdmin):
 
 
 class CustomUserAdmin(UserAdmin):
-    add_form = CustomUserRegistrationForm
+    add_form = UserCreationForm
```

Comment:
Why are we no longer using `CustomUserRegistrationForm` here? I'm surprised that it still exists even though we're removing it here.